rhp/v2/rhp.go

```diff
@@ -280,45 +303,116 @@ type (
 	}
 )
 
-// RPCSectorRootsCost returns the price of a SectorRoots RPC.
-func RPCSectorRootsCost(settings HostSettings, n uint64) types.Currency {
-	return settings.BaseRPCPrice.
-		Add(settings.DownloadBandwidthPrice.Mul64(n * 32)).  // roots
-		Add(settings.DownloadBandwidthPrice.Mul64(128 * 32)) // proof
+type RPCCost struct {
```

Comment:
Functionally this type matches the `ResourceCost` in RHPv3, we could move this into the `types` package maybe and use the one type. I didn't want to do that yet mostly because of the documentation in `ResourceCost` which is tailored to the MDM... Maybe it's fine to just have both, for the same reason `hostd` had its own internal `rpcCost` type.